code/modules/assembly/flash.dm

```diff
@@ -126,7 +126,7 @@
 				to_chat(M, "<span class='userdanger'>[user] blinds you with the flash!</span>")
 			else
 				to_chat(M, "<span class='userdanger'>You are blinded by [src]!</span>")
-			M.Paralyze(rand(80,120))
+			M.Knockdown(rand(80,120))

```

Comment:
can you remove funny randomness

Reply:
I'm not entirely against it, like to hear what others say.

Reply:
I think an important aspect of the flash is its unreliability, ranging from its unpredictable burnout to its semi random effects. I don’t know if it would actually matter if you changed this, just make sure you’re consciously doing so.